rncryptor.cabal

```diff
@@ -31,8 +31,8 @@ library
     , random >= 1.0.0.1
     , QuickCheck >= 2.6 && < 2.9
     , io-streams >= 1.2.0.0
-    , cipher-aes >= 0.2.9
-    , pbkdf >= 1.1.1.1
+    , cryptonite
```

Comment:
I'm not opposed to switch to `cryptonite` as the default crypto package (after all, Vincent created it to replace the previous ones he wrote), but being unbound in its dependency is a bit risky, as we can easily stumble upon bugs. For example, I strongly recommend having as a lower bound at least `>= 0.15`, to avoid this:

https://github.com/vincenthz/hs-tls/issues/124#issuecomment-207867088

And probably have an upper bound to `0.21`, so we cut ourselves some slack when version `0.20` would be released.